spec/acceptance/sessions/sign_in_spec.rb

```diff
@@ -0,0 +1,46 @@
+# frozen_string_literal: true
+
```

Comment:
We should stick to request tests. @JulianPasquale wdyt?

Reply:
Yeah, agree

Reply:
@jackrootstrap Can you address this comment, please?

Reply:
This was already addressed with @JulianPasquale offline!